style.css

```diff
@@ -0,0 +1,129 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@400;500;600;700&family=Poppins:wght@300;400;500&family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,700&display=swap");
+
+:root {
+  --theme-color: #1c1a19;
+  --toolbar-color: #3c3a39;
+  --primary-1: #ff6b00;
+  --primary-4: #fff5e1;
+  --neutral: #dbd8d7;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+header {
+  z-index: 1;
+  background: var(--toolbar-color);
+  width: 100%;
+  height: 92px;
+  top: 0;
+}
+
+.nav-bar {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 12px;
+}
+
+.nav-bar h3 {
+  position: absolute;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+  color: #fff;
+  width: 101px;
+  height: 44px;
+  left: 12px;
+  top: 50%;
+  transform: translateY(10%);
+}
+
+.nav-bar img {
+  position: absolute;
+  padding: 8px;
+  width: 40px;
+  height: 40px;
+  right: 24px;
+  top: 50%;
+  transform: translateY(30%);
+}
```

Comment:
- [ ] Properties listed below are also a problem to the design of the page. Consider removing them, look at the result, then make relevant adjustments.

```CSS
    position: ;
    line-height: ;
    width: ;
    height: ;
    left: ;
    top: ;
    transform: ;
```

- [ ] The design of the page should match the provided Figma design. See the comparison below. There is a huge difference. When you are done making changes, please test your page in mobile view.

### Implementation
![implementation](https://user-images.githubusercontent.com/45002654/124993968-c65cb480-e04d-11eb-9398-6d97b75fd954.PNG)

### The Figma Design
![design](https://user-images.githubusercontent.com/45002654/124993991-d2487680-e04d-11eb-90a1-f6c832a6d8b5.PNG)